### ISSUE TITLE
Bug: User autocomplete doesn't update with newly added users

### ISSUE THREAD

**Issue by chenilim** [mattermost/focalboard#2392]
## Steps to reproduce the behavior

1. Create a board off a private channel
2. Create and open a card in the board
3. In a separate tab, add user2 to the private channel
4. In the original tab, type `@` in a comment or description to see the list of users
5. User2 does not appear in the list (until a Cmd+R refresh)

## Expected behavior

The auto-complete should show the users who have access to the board.

## Edition and Platform

 - Edition: Mattermost plugin
 - Version: v0.14
 - Browser and OS: Chrome on Mac

## Additional context

We should ensure that this works even after the permissions changes, i.e.
* Typing `@` should show the list of users who have access to the board
* For boards that are public to the team, all users in the team should appear in the list

**Comment by chenilim** [comment]
Retest on permissions branch.

**Comment by chenilim** [comment]
Retested on bugbash2:
1. Open a card in a private board as user1 on browser tab1
2. Open the same board as user1 in a separate tab (tab2), and add user2 to it
3. Back in tab 1, type @ in a comment. User2 appears in the list

So, this specific issue appears to be resolved. Closing this, but I've opened https://github.com/mattermost/focalboard/issues/3002 to track the related issue of removed members not reflected in the autocomplete list.
